Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,27 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+
+namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
+            string[] stats = new string[4];
             // Print each of the statistical output using Text Table with padding 3:
             // - FindHighestBalanceEver
+            stats[0] = BalanceStats.FindHighestBalanceEver(PeoplesBalances.Balances);
             // - FindPersonWithBiggestLoss
+            stats[1] = BalanceStats.FindPersonWithBiggestLoss(PeoplesBalances.Balances);
             // - FindRichestPerson
+            stats[2] = BalanceStats.FindRichestPerson(PeoplesBalances.Balances);
             // - FindMostPoorPerson
+            stats[3] = BalanceStats.FindMostPoorPerson(PeoplesBalances.Balances);
+
+            int padding = 3;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'padding'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5069037)